packages/mrgn-ui/src/components/action-box-v2/actions/deposit-swap-box/deposit-swap-box.tsx

```diff
@@ -638,7 +475,7 @@ export const DepositSwapBox = ({
       <Preview
         actionSummary={actionSummary}
         selectedBank={selectedDepositBank}
-        isLoading={isLoading}
+        isLoading={isSimulating.isLoading}
```

Comment:
isSimulating implies a boolean type, simulationStatus would perhaps make it a bit more clear